views/setting/page-select.php

```diff
@@ -7,8 +7,8 @@
 
 ?>
 <div class="item-description">
-	<h3><?php echo esc_html( $prpl_setting['title'] ); ?></h3>
-	<p><?php echo esc_html( $prpl_setting['description'] ); ?></p>
+	<!-- <h3><?php echo esc_html( $prpl_setting['title'] ); ?></h3>
+	<p><?php echo esc_html( $prpl_setting['description'] ); ?></p> -->
```

Comment:
Cleanup
```suggestion
```

Reply:
If we add radios into the page-select control, we will need this